lang/src/org/partiql/lang/eval/physical/operators/PhysicalOperatorFactory.kt

```diff
@@ -0,0 +1,9 @@
+package org.partiql.lang.eval.physical.operators
+
+/**
+ * Marker interface with unique [key], allowing all [PhysicalOperatorFactory] implementations to exist in a
+ * `Map<OperatorFactoryKey, OperatorFactory>`.
+ */
+interface PhysicalOperatorFactory {
```

Comment:
> Can we rename this class to PhysicalOpeator and rename create method to eval? 

But it doesn't evaluate anything, hence `eval` would *not* be the right name.  It creates a `BindingsExpr` that *can* do the evaluation.  Hence the terms `create` and `Factory`.  I'm open to changing these terms, but not to `eval`.

Reply:
What about `PhysicalOperator` with `invoke`?

Reply:
`PhysicalOperatorFactory` doesn't actually evaluate anything.  What it does is create the thing that actually does the evaluation.  Therefore, it would not be appropriate to rename it to `PhysicalOperator`, nor to rename `create` to `eval` or `invoke`.    I have some new names for these items in the next commit I'm about to push--hopefully this will become clearer then.